### DISCUSSION TITLE
Jetstream now broken Upon Upgrade, Alias Emit()?

### DISCUSSION THREAD

**Discussion by Jafo232** [livewire/livewire#6032] [1. Help] [answered]
Is there any way to just alias the emit() method to make things like Jetstream backwards compatible?

Right now I get these errors because in the package files there still is:

![image](https://github.com/livewire/livewire/assets/1456061/b899cf97-cd23-4216-972b-3a547ba0e94d)

![image](https://github.com/livewire/livewire/assets/1456061/b7a396ed-f576-4cd0-87de-fe499f54e7a0)

**Comment by PhiloNL** [comment, marked as answer]
You would have to find & replace these yourself, as Jetstream is not yet compatible at this time. The Laravel team will make the required changes when the stable release approaches. 

Alternatively, you could try a macro:

```php
Component::macro('emit', function ($event) {
    $this->dispatch($event);
});
```

**Reply by Jafo232** [reply]
Thanks.  For now, I have just folded Jetstream into my source as an internal package and updated it until they come out with a compatible version.
